kola/tests/kubernetes/kubelet_wrapper.go

```diff
@@ -63,10 +63,10 @@ func testKubeletWrapperVarLog(c cluster.TestCluster) {
 	c.MustSSH(m, `
 	for i in {1..120}; do 
 		sleep 5
-		if journalctl -u kubelet -o cat | grep '`+versionOutput+`' &>/dev/null; then
+		if journalctl -t kubelet-wrapper -o cat | grep '`+versionOutput+`' &>/dev/null; then
```

Comment:
While we're modifying this, I think it would be nice to use `grep -F` since the pattern contains regex characters.  It shouldn't matter with this specific case, but it will make things look a bit saner.

Reply:
good idea. Fixed.